src/AdminConsole/Services/Mail/DefaultMailService.cs

```diff
@@ -62,12 +63,17 @@ public async Task SendEmailIsAlreadyInUseAsync(string email)
 
     public Task SendMagicLinksDisabledAsync(string organizationName, string email)
     {
+        if (!EmailAddressValidator.IsValid(email))
```

Comment:
Changing the method signature to take a `System.Net.Mail.MailAddress` instead of a `string` would make the method easier to call correctly and give us the added benefit of built in validation of the email.